jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -20,7 +20,11 @@ If you do not have a `circle.yml` file, refer to the [Sample 2.0 `config.yml` Fi
 
 CircleCI 2.0 introduces the requirement that you create a configuration file (`.circleci/config.yml`), and it adds new required keys for which values must be defined. This release also allows you to use multiple jobs in your configuration. **Note:** Parallelism in 2.0 can only be set in `.circleci/config.yml`, the parallelism setting from the UI is ignored.
 
-If you already have a `circle.yml` file, the following sections describe how to make a copy of your existing file, create the new required keys, and then search and replace your 1.0 keys with 2.0 keys. It is also possible to use the beta `config-translation` endpoint to get started with converting a 1.0 config to 2.0, see [Using the 1.0 to 2.0 config-translation Endpoint]({{ site.baseurl }}/2.0/config-translation).
+If you already have a `circle.yml` file, the following sections describe how to make a copy of your existing file, create the new required keys, and then search and replace your 1.0 keys with 2.0 keys.
+
+## Using the 1.0 to 2.0 config-translation Endpoint
```

Comment:
Something about "the 1.0 to 2.0 config-translation Endpoint" feels less clear than it could be. I know it's just an endpoint, but maybe something like "The CircleCI Config Migration Tool" would be more obvious? Up to you @iynere.

Reply:
"`config-translation` endpoint" is the official name 🤷‍♀️ feel free to make it something else if you think that's better (i agree w/ you fwiw lol)